app/views/support/CountryWithCurrency.scala

```diff
@@ -5,12 +5,21 @@ import com.gu.i18n.{Country, CountryGroup, Currency}
 
 import scala.language.implicitConversions
 
-case class CountryWithCurrency(country: i18n.Country, currency: i18n.Currency)
+sealed trait StripeServiceName {
+  val jsLookupKey: String
+}
+case object UKStripeService extends StripeServiceName {
+  val jsLookupKey = "ukPublicKey" // Referenced in assets/javascripts/modules/checkout/submit.js
+}
+case object AUStripeService extends StripeServiceName {
+  val jsLookupKey = "auPublicKey"
```

Comment:
Is this used?

Reply:
Line 43 of javascriptFirstSteps.scala.html

Reply:
Fixed